scripts/collect.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python3
+from math import log, tan, pi
+from itertools import product
+from argparse import ArgumentParser
+from os.path import join
+
+import tempfile, shutil, urllib.request, io, sys, subprocess
+import unittest, unittest.mock as mock
+
+tile_url = 'https://terrain-preview.mapzen.com/geotiff/{z}/{x}/{y}.tif'
+
+def mercator(lat, lon, zoom):
+    ''' Convert latitude, longitude to z/x/y tile coordinate at given zoom.
```

Comment:
If the GDAL python bindings are available, is it possible to use that to [reproject](https://pcjericks.github.io/py-gdalogr-cookbook/projection.html#reproject-a-geometry)? Although that does look pretty convoluted.

I couldn't find a pure Python package of projections (try saying that 3 times quickly! :laughing:), does that perhaps imply that such a thing would be more generally useful than just here?